Procfile

```diff
@@ -1 +1 @@
-web: node src/
\ No newline at end of file
+web: NODE_ENV=production node src/
```

Comment:
Ah, I usually set this in the environment configuration but this makes much more sense.